equinix/resource_network_device.go

```diff
@@ -112,6 +113,7 @@ var neDeviceDescriptions = map[string]string{
 	"ClusterDetails":      "An object that has the cluster details",
 	"ValidStatusList":     "Comma Separated List of states to be considered valid when searching by name",
 	"Connectivity":        "Parameter to identify internet access for device. Supported Values: INTERNET-ACCESS(default) or PRIVATE or INTERNET-ACCESS-WITH-PRVT-MGMT",
+	"ProjectID":           "Unique identifier of Resource",
```

Comment:
Unique identifier of Project might be more accurate.

Reply:
addressed